src/NovaS3MultipartUpload.php

```diff
@@ -528,6 +529,17 @@ public function displayPoweredByUppy($boolean = true)
         return $this->withMeta(['proudlyDisplayPoweredByUppy' => $boolean]);
     }
 
+    /**
+     * View resource when user press click at the file icon.
+     *
+     * @param  bool  $boolean
+     * @return $this
+     */
+    public function viewResource($boolean = true)
```

Comment:
Better to name it `linkToResource` and you can check here if the relationship model has a corresponding Nova resource